### ISSUE TITLE
12/1 - Tact AI missed a key oppurtunity to do a crippling road pillage

### ISSUE THREAD

**Issue by Stalker012345** [LoneGazebo/Community-Patch-DLL#5932]
_1. Mod version (i.e Date - 4/23):_
12/1

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
See Screenshot. On the last turn that Fusilier was right next to my city, and had a perfect chance to double pillage and take out my road. That would have disconnected over half of my empire, it would have been a major hit to me in this war...but the AI just moved on with no pillage at all.

_4. Steps to reproduce (optional):_

![image](https://user-images.githubusercontent.com/20593976/70209070-df281a00-16f4-11ea-834c-f22866ab3f8c.png)

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by ilteroi** [comment]
there are no high-level strategic considerations for pillaging right now, that unit probably just wanted to flee ...

**Comment by bigcat88** [comment]
No, pillaging in that situation do nothing. Lighthouses will continue connections.
